src/App.js

```diff
@@ -17,18 +17,34 @@ function App() {
     navigate('/', { replace: true });
   }
 
+  function handleDelete(id) {
+    const newData = storageData.filter(storeddata => storeddata.id !== id);
+    setStorageData(newData);
+    console.log('ich wurde aufgerufen');
```

Comment:
🕵️ it is good practice to remove all console.logs when  creating a PR.

Reply:
thanks, yes, you are absolutely right. I forgot to remove this